sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -246,6 +247,8 @@ def _create_pipeline(self, credential, **kwargs):
             DistributedTracingPolicy(**kwargs),
             HttpLoggingPolicy(**kwargs)
         ]
+        if kwargs.get("_additional_pipeline_policies"):
+            policies = policies + kwargs.get("_additional_pipeline_policies")
```

Comment:
wow does '+' function as append here?

Reply:
Yes.

Reply:
Let's copy it to file-share, file-datalake and queue's base_client